promotion.md

```diff
@@ -105,12 +95,14 @@ extension but actually does not:
 const EMPTY_BYTES: &Vec<u8> = &Vec::new(); // Ok without lifetime extension
 ```
 
-As we have seen above, `Vec::new()` does not get promoted. And yet this
+`Vec::new()` cannot get promoted because it needs dropping. And yet this
 compiles. Why that? The reason is that the reference obtains the lifetime of
 the "enclosing scope", similar to how `let x = &mut x;` creates a reference
 whose lifetime lasts for the enclosing scope. This is decided during MIR
 building already, and does not involve lifetime extension.
 
+This is also sometimes called "outermost scope" rule.
```

Comment:
@eddyb calls it "outermost scope" rule. I am not sure why. I will try to find out.